workflows/variantconsensus.nf

```diff
@@ -16,73 +28,239 @@ include { methodsDescriptionText } from '../subworkflows/local/utils_nfcore_vari
 */
 
 workflow VARIANTCONSENSUS {
-
     take:
     ch_samplesheet // channel: samplesheet read in from --input
+
     main:
 
     ch_versions = Channel.empty()
+    ch_reports = Channel.empty()
     ch_multiqc_files = Channel.empty()
 
+    ch_both = ch_samplesheet.filter { meta, _files ->
+            meta.varianttype != 'snps' && meta.varianttype != 'indels'
+        }
+
+    //
+    // SNP WORKFLOW
+    //
+
+    ch_snps = ch_samplesheet.filter { meta, _files ->
+            meta.varianttype == 'snps'
+        }
+
+    // FILTER_SNPS to divide into SNPs and INDELs
+    FILTER_SNPS(
+        ch_both.map { meta, vcfs -> [meta, vcfs[0], vcfs[1]] },
+        [],
+        [],
+        [],
+    )
+
+    ch_divided_snps = FILTER_SNPS.out.vcf.join(FILTER_SNPS.out.tbi)
+            .map { meta, vcf, tbi -> [ meta.subMap(meta.keySet() - ['varianttype']) + [ 'varianttype': 'snps' ], [vcf, tbi] ] }
+
+    ch_versions = ch_versions.mix(FILTER_SNPS.out.versions)
+
+    // Group SNPs for ISEC
+    ch_snps_grouped = Channel.empty()
+        .mix(ch_snps, ch_divided_snps)
+        .map { meta, files ->
+            [ [meta.id, meta.sample, meta.varianttype], meta, files[0], files[1] ]}
+        .groupTuple(by: [0])
+        .map { _id, metas, vcfs, tbis ->
```

Comment:
Just a question here: Do you just access the `meta.id` with the `_id` or what does the underscore mean here? Have not seen this before.